client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx

```diff
@@ -103,6 +105,7 @@ const ProcessingStep: Step = function ( props ): ReactElement | null {
 			}
 			stepProgress={ stepProgress }
 			recordTracksEvent={ recordTracksEvent }
+			showJetpackPowered={ isJetpackPowered }
```

Comment:
This will add the jetpack powered to all the flows I think? Not important now, we're adding specialized processing steps for the new flows 👌 You can remove this.

Reply:
I was thinking to reuse the processing step instead of building a new one.
The boolean `isJetpackPowered` is true only for the newsletter flow 
https://github.com/Automattic/wp-calypso/pull/66526/files#diff-b7ff3243075883b8ecc07a8d6f6e9c94a960ec6d179ac29676a74d6270a6fe1aR79

Reply:
Oh well, we have different designs and everything, @renancarvalho is working on that so no need here 👍